js/deer-utils.js

```diff
@@ -167,11 +167,31 @@ export default {
      * Execute query for any annotations in RERUM which target the
      * id passed in. Promise resolves to an array of annotations.
      * @param {String} id URI for the targeted entity
+     * @param [String] targetStyle other formats of resource targeting.  May be null
      */
-    findByTargetId: async function (id) {
+    findByTargetId: async function (id, targetStyle) {
         let everything = Object.keys(localStorage).map(k => JSON.parse(localStorage.getItem(k)))
-        let obj = {
-            target: id
+        //Backwards compatible without the new parameter
+        if(targetStyle === undefined){
+            targetStyle = []
+        }
+        //Users pass the craziest things...
+        if (!Array.isArray(targetStyle)) {
+            if(typeof targetStyle === "string"){
+                targetStyle = [targetStyle]
+            }
+            else{
```

Comment:
The comment is fine, but `targetStyle` here may be an object or number, so that can be dropped into an array without breaking anything. I wouldn't blank an Array without at least noting the major error in what the dev intended to send in and what is being processed.

tl;dr : remove everything but the original `if`